google-cloud-datastore/src/test/java/com/google/cloud/datastore/DatastoreTest.java

```diff
@@ -231,6 +232,8 @@ public void testNewTransactionCommit() {
     verifyNotUsable(transaction);
   }
 
+  // TODO(gapic_upgrade): Remove the @ignore annotation
+  @Ignore("This should be fixed with actionable error implementation")
```

Comment:
please file a github issue tracking all of the ignored tests so we don't lose track

Reply:
#1242 created